WSLBlobNFS/wsl2_linux_script.sh

```diff
@@ -63,17 +144,22 @@ function onetime_samba_setup ()
 # To-do: Return the result of the mount command
 function mount_nfs ()
 {
-    echo "Got $# number of args: $*"
+    # vecho "Got $# number of args: $*"
     # execute the mount command passed as the first argument
     $1
 
+    if [[ $? != 0 ]]; then
+        eecho "Failed to mount NFS share with: $1"
+        exit 1
+    fi
+
     # Set the read ahead to 16MB
-    echo "Setting read ahead to 16MB"
+    vecho "Setting read ahead to 16MB"
     cd $2
-    echo 16384 > /sys/class/bdi/0:$(stat -c "%d" .)/read_ahead_kb
+    vecho 16384 > /sys/class/bdi/0:$(stat -c "%d" .)/read_ahead_kb
     cd -
```

Comment:
instead of doing cd and back, just use

echo 16384 > /sys/class/bdi/0:$(stat -c "%d" $2)/read_ahead_kb

also note, don't use vecho as it may emit some escape characters